### ISSUE TITLE
Possibility to add selected a single date or range programatically

### ISSUE THREAD

**Issue by Xpjay** [JoshHalvorson/calendar-date-range-picker#8]
Hi,

I think it would be interesting, when I open the calendar I would like the possibility to selected a date or range by default.
I have buttons like "today", "last week" "last month" .... but can't access to any fonction to preselect the date by passing calendar object

Something like:

`calendarDateRangePicker.setSelectedDate(today)` if I want to select today

`calendarDateRangePicker.setSelectedDate(lastWeek, today)` if I want to select the last week as range

**Comment by JoshHalvorson** [comment]
Yeah I think that'd be a useful feature, will add that to the todo, thanks!
